clients/java/src/test/java/marquez/client/MarquezClientTest.java

```diff
@@ -479,7 +486,7 @@ public void testCreateDbTable() throws Exception {
             .build();
 
     final String metaAsJson = JsonGenerator.newJsonFor(meta);
-    final String dbTableAsJson = JsonGenerator.newJsonFor(DB_TABLE);
```

Comment:
I don't see any reason to use that class

Reply:
but the cons:
 - It should be extended at any model change (like my case) vs automatic mapping via ObjectMapper
 - Additional class

Reply:
The `JsonGenerator.newJsonFor()` is similar to defining a json file that you load in your tests that represents the serialized representation of the object that you want to deserialized, then assert on. If you just use the `ObjectMapper`, yeah you don't have to worry about defining json (via `JsonGenerator.newJsonFor()` or a json file), but here you're just confirming that the `ObjectMapper` result is equal to the string returned by calling the models `fromJson()` method.

The idea behind `JsonGenerator.newJsonFor()` was to use the objects generated by our `ModelGenerators` to ensure the json has valid values. We can move to defining json files for our models, but that also means the author of the test has to define the values. Maybe that's not the worst thing.

That said, I would still keep `JsonGenerator.newJsonFor()`, but instead of having a specific type (`Dataset`, `Job`, etc), I would just wrap the mapper call to serialize:

```java
public static String newJsonFor(final Object o) {
    return MAPPER.writeValueAsString(o);
}
```

Then, you can delete the other methods in [`JsonGenerator`](https://github.com/MarquezProject/marquez/blob/main/clients/java/src/test/java/marquez/client/models/JsonGenerator.java). I think this would help with overall test maintenance.

Reply:
I leave it and add field there